client/my-sites/plugins/marketplace-footer/styles.module.css

```
@@ -0,0 +1,37 @@
+.marketplace-footer {
+	--color-accent: var( --studio-blue-50 );
+	--color-accent-60: var( --studio-blue-60 );
+	margin-bottom: -32px;
+
+    & .marketplace-footer__section::before {
+        background-color: #f6f7f7;
```

Probably meant to be this and not a magic number:

```suggestion
        background-color: var(--studio-gray-0);
```

---

> Probably meant to be this and not a magic number:

Updated in 1e69f46d532.